File: sponge-jython/examples/script/py/hello_world.py
"""
Sponge Knowledge base
Hello world
"""

class HelloWorldTrigger(Trigger):
    def onConfigure(self):
        self.event = "helloEvent"
    def onRun(self, event):
        print event.get("say")

def onStartup():
    EPS.event("helloEvent").set("say", "Hello World!").send()

File: sponge-jruby/examples/script/rb/hello_world.rb
# Sponge Knowledge base
# Hello world

class HelloWorldTrigger < Trigger
    def onConfigure
        self.event = "helloEvent"
    end

    def onRun(event)
        puts event.get("say")
    end
end

def onStartup
	$EPS.event("helloEvent").set("say", "Hello World!").send()
end

